va/nfv/ovs-dpdk-sriov/edpm/kustomization.yaml

```diff
@@ -24,6 +24,14 @@ resources:
   - baremetalset-password-secret.yaml
   - nova_ovs_dpdk_sriov.yaml
 
+secretGenerator:
```

Comment:
This section is already covered:
https://github.com/openstack-k8s-operators/architecture/blob/main/lib/dataplane/nodeset/kustomization.yaml#L10-L16

I think this PR is not required.

Reply:
It was missing when we tried to do some changes manually we should close this pr